app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -9,4 +9,64 @@ def google_oauth2
       redirect_to new_user_registration_url, alert: @user.errors.full_messages.join("\n")
     end
   end
+
+  def trello
+    auth = request.env["omniauth.auth"]
+    @identity = Identity.all.find_by(:uid => auth.uid, :provider => auth.provider, :user_id => current_user.id)
+    if @identity.nil?
+      @identity = Identity.create(:uid => auth.uid, :provider => auth.provider, :trello_auth_token => auth.extra.access_token.token, :trello_auth_secret => auth.extra.access_token.secret, :user_id => current_user.id)
```

Comment:
it looks like we have a duplication here. We have `auth.uid` in `Identity` and in `User` models.